spec/intl.html

```diff
@@ -640,6 +813,40 @@ <h1>Intl.DateTimeFormat.prototype.formatToParts ( _date_ )</h1>
       </emu-alg>
     </emu-clause>
 
+    <emu-clause id="sec-intl.datetimeformat.prototype.formatRange">
+      <h1>Intl.DateTimeFormat.prototype.formatRange ( _startDate_, _endDate_ )</h1>
+
+      <p>
+        When the `formatRange` method is called with an arguments _startDate_ and _endDate_, the following steps are taken:
+      </p>
+
+      <emu-alg>
+        1. Let _dtf_ be *this* value.
+        1. Perform ? RequireInternalSlot(_dtf_, [[InitializedDateTimeFormat]]).
+        1. If _startDate_ is *undefined* or _endDate_ is *undefined*, throw a *TypeError* exception.
+        1. <del>Let _x_ be ? ToNumber(_startDate_).</del>
+        1. <del>Let _y_ be ? ToNumber(_endDate_).</del>
+        1. Return ? FormatDateTimeRange(_dtf_, <del>_x_</del><ins>_startDate_</ins>, <del>_y_</del><ins>_endDate_</ins>).
+      </emu-alg>
+    </emu-clause>
```

Comment:
Rather than `<del>...<ins>` here, this needs to be fixed in [the source](https://tc39.es/proposal-intl-DateTimeFormat-formatRange/#sec-intl.datetimeformat.prototype.formatRange), where there's an error in the names (`_x_, _y_` used but not previously defined) but it is otherwise identical.

Reply:
Disregard, misread